_studio/mfx_lib/shared/src/mfx_vpx_dec_common.cpp

```diff
@@ -472,7 +472,7 @@ namespace MFX_VPX_Utility
         // Increase minimum number by one
         // E.g., decoder unlocks references in sync part (NOT async), so in order to free some surface
         // application need an additional surface to call DecodeFrameAsync()
-        p_request->NumFrameMin += 1;
+        p_request->NumFrameMin += 2;
```

Comment:
I have a concern for system memory path where we have p_request->NumFrameSuggested != p_request->NumFrameMin:

https://github.com/Intel-Media-SDK/MediaSDK/blob/52c71f460abea29fce1dea69df2b67c8d869a80e/_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp#L554

I suspect this is a bug and we need to have these 2 values equal to each other. Can you, please, review this place.